osmaxx/excerptexport/templates/excerptexport/email/all_exports_of_extraction_order_done_subject.txt

```diff
@@ -0,0 +1 @@
+Extraction Order #{{ extraction_order.id }} "{{ extraction_order.excerpt_name }}": {{ successful_exports_count }} Export{{ successful_exports_count|pluralize }} finished successfully, 1 failed
```

Comment:
oops, missed to parametrize completely that after copy-pasting the expected string for the test

now done in 3819fdecc2450c3419602b4323fa7a6f6f576fdc